File: src/main/java/plugin/command/owner/ReloadDoorsCommandPlugin.java
package plugin.command.owner;

import org.redrune.game.content.plugin.type.CommandPlugin;
import org.redrune.game.entity.actor.player.Player;
import org.redrune.utility.game.repository.object.door.DoorRepository;
import plugin.command.CommandManifest;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 2019-02-20
 */
@CommandManifest(description = "Reloads all doors from file")
public class ReloadDoorsCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
		DoorRepository.initialize();
		player.getPackets().sendMessage("Reloaded all game doors");
	}
	
	@Override
	public String[] identifiers() {
		return arguments("rldrs", "rldoors");
	}
}

File: src/main/kotlin/plugin/command/owner/ReloadDoorsCommandPlugin.kt
package plugin.command.owner

import org.redrune.game.content.plugin.type.CommandPlugin
import org.redrune.game.entity.actor.player.Player
import org.redrune.utility.game.repository.door.DoorRepository
import plugin.command.CommandManifest

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 2019-02-20
 */
@CommandManifest(description = "Reloads all doors from file")
class ReloadDoorsCommandPlugin : CommandPlugin() {

    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        DoorRepository.initialize()
        player.packets.sendMessage("Reloaded all game doors")
    }

    override fun identifiers(): Array<String> {
        return arguments("rldrs", "rldoors")
    }

}